wgengine/magicsock/magicsock.go

```diff
@@ -487,6 +497,8 @@ func NewConn(opts Options) (*Conn, error) {
 		GetSTUNConn4:        func() netcheck.STUNConn { return c.pconn4 },
 		SkipExternalNetwork: inTest(),
 	}
+	c.netChecker.SetPortMapper(c.portMapper)
```

Comment:
Rather than a separate setter, can you pass it in the construction of netcheck.Client above? It seems we only ever want netcheckers with "don't care" portmappers or a specific portmapper for the lifetime of the netchecker, which seems like a better fit for construction params.